contributing-device-support.md

```diff
@@ -1,19 +1,25 @@
-# Custom Board Support
+# Customer Board Support
+
+There are several options for onboarding the hardware onto balenaOS:
+
+1. You can find another board that would work for your case that is supported in our [supported device list](https://www.balena.io/docs/reference/hardware/devices/).
+2. You can consider our [Custom Device Support (CDS)](https://www.balena.io/pricing/#custom-device-support), which is a paid service during which we onboard the device for you and maintain it for a monthly fee. [Here](https://drive.google.com/file/d/17-DTeXkflkzW8Qduj48ut7JNUTWqiRv-/view?usp=sharing) is an overview including pricing.
+3. If you have folks on your team who are familiar with Yocto, then you could build and maintain it yourselves using our customer board support (CBS) documentation. Follow along below!
```

Comment:
```suggestion
3. Building and maintaining balenaOS yourselves using our Customer Board Support (CBS) documentation. This will require knowledge of the Yocto project and familiarity with tools used to build custom images. Follow along the documentation for CBS below: 
```